manager/pom.xml

```diff
@@ -30,6 +30,14 @@
             <artifactId>infinispan-extension</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
We should also add the service-binding extension:

```xml
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-kubernetes-service-binding</artifactId>
        </dependency>
```

Reply:
As we're using the reactive client I don't think any additional implementation is required, just the dependency, so I think it makes sense to include it now.